ml_engine/online_prediction/requirements.txt

```
@@ -1,5 +1,5 @@
-tensorflow==2.0.1; python_version > "3.0"
-tensorflow==1.15.0; python_version < "3.0"
+tensorflow==2.7.0; python_version > "3.0"
+tensorflow==2.7.0; python_version < "3.0"
```

```suggestion
tensorflow==2.6.2; python_version < "3.0"
```
latest available version is 2.6.2